src/service/stockService.js

```diff
@@ -44,5 +44,18 @@ export default class StockService {
 
         return stockWithUsers;
     }
+
+    async postCandlestickData(stockData) {
+        try {
+            var ticker = await this.stockRepository.getStockTicker(stockData.Ticker)
+            if (undefined == ticker) {
+                await this.stockRepository.postStockMetadata(stockData.Ticker, stockData.CompanyName);
+            }
+            await this.stockRepository.postCandlestickData(stockData);
+        } catch (ex) {
+            this.logger.error({Ticker: stockData.Ticker}, 'Error when looking at the following ticker.');
```

Comment:
this catch also includes posting candlestick data. The message in the logger.error is a little misleading

Reply:
Done